pkg/controller/controller_instance.go

```diff
@@ -1545,12 +1545,19 @@ func (c *controller) prepareServiceInstanceLastOperationRequest(instance *v1beta
 		return nil, err
 	}
 
+	// allow for nil ServicePlan which happens if user sets to non-existing plan
```

Comment:
I wasn't expecting changes to this file? Was this intentional? If so, I think it should be in a separate PR with more info on what it's fixing.

Reply:
nope, that definitely is a bleed over from another PR, good catch, thanks much @carolynvs !